src/openms/include/OpenMS/ANALYSIS/ID/SiriusMSConverter.h

```diff
@@ -34,12 +34,12 @@
 
 #pragma once
 
-#include <OpenMS/DATASTRUCTURES/String.h>
-#include <OpenMS/DATASTRUCTURES/StringListUtils.h>
 #include <OpenMS/KERNEL/StandardTypes.h>
```

Comment:
I dont think you need any of these includes except `StandardTypes.h` - you can forward declare the rest, right?

Reply:
That is right!